editor.planx.uk/src/pages/FlowEditor/components/Settings/ServiceSettings.tsx

```diff
@@ -280,6 +421,14 @@ const ServiceSettings: React.FC = () => {
             </p>
             <p>Offline services can still be edited and published as normal.</p>
           </SettingsDescription>
+
+          <PublicLink
+            isFlowPublished={isFlowPublished}
+            status={statusForm.values.status}
```

Comment:
This might be the required fix here?

```suggestion
            status={flowStatus}
```

Reply:
@DafyddLlyr I may have missed something, what is it fixing? I think this is the change we chatted about on Slack?

Reply:
Yep - this is it - I linked to this comment in the Slack chat, just wanted a note here also linked to the specific line.

<img width="844" alt="image" src="https://github.com/user-attachments/assets/e6ee7140-0e92-4bb1-ab42-3e62e5866935">

Apologies for any confusion, please mark as resolved if sorted! 👍